php-classes/Slate/CBL/Demonstrations/Demonstration.php

```diff
@@ -53,6 +54,12 @@ class Demonstration extends \VersionedRecord
             'class' => DemonstrationSkill::class,
             'foreign' => 'DemonstrationID',
             'prune' => 'delete'
+        ],
+        'StudentTask' => [
+          'type' => 'one-one',
+          'class' => StudentTask::class,
+          'local' => 'ID',
+          'foreign' => 'DemonstrationID'
```

Comment:
@BillClinton one tweak to add here: there is no unique constraint on on the `DemonstrationID` column in the `StudentTask` table, so theoretically there can be multiple `StudentTask` records pointing to the same `Demonstration`. 

We should either add a unique constraint to the `DemonstrationID` column, or adjust this relationship config to at least make it deterministic in the case of multiple matches existing

```suggestion
          'foreign' => 'DemonstrationID',
          'order' => ['ID' => 'DESC']
```

would suffice to at least make sure the newest match is always returned

Reply:
@BillClinton I went ahead and added the new unique constraint in 02e2cd07fb2c8fe71d90064ca0f590d0617701d6

Reply:
so just read this for FYI but then mark resolved, no need to add the order but FYI that you can do that